docs/content/en/docs/installation-configuration/authentication.md

```diff
@@ -0,0 +1,253 @@
+
+---
+
+title: Setup RBAC for SOARCA
+description: >
+  Setup OIDC based Role-Based Access Control (RBAC) for SOARCA
+categories: [extensions, architecture]
+tags: [security]
+weight: 2
+---
+
+Authentication is disabled by default in SOARCA. This means that when SOARCA is launched with its default configuration and exposed to a network, anyone can interact with it. Since SOARCA requires significant capabilities and access to reconfigure systems, exposing it without authentication poses a security risk. This section outlines how to set up authentication and authorization for SOARCA.
+
+SOARCA leverages our internally developed [gauth library](https://github.com/COSSAS/gauth) as its underlying authentication framework. This library provides convenient Role-Based Access Control (RBAC) middleware, which manages authentication for various endpoints, such as the Playbook API. Based on OpenID Connect (OIDC), the library supports integration with multiple authentication providers.
+
+{{% alert title="Warning" color="warning" %}}
+Currently, [gauth library](https://github.com/COSSAS/gauth) is supported and tested authentication for the [Authentik](https://goauthentik.io/) authentication provider, an open-source solution that supports a wide range of authentication methods. As such, other OIDC-based providers might not be compatible.
+{{% /alert %}}
+
+{{% alert title="Note" color="primary" %}}
+For the current version, users/application must belong to the `soarca_admin` group to access the API's when authentication is enabled. Furthermore, currently there is not yet control over what user can go to which route. This feature is scheduled for a future version of SOARCA.
+{{% /alert %}}
```

Comment:
This is confusing make the group configurable and make a list of supported providers